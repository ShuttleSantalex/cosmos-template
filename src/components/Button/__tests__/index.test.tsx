import React from 'react';
import { expectToMatchSnapshot } from '../../../utils/match-snapshot';

import { Button } from '..';

describe('Button', () => {
  it('matches snapshot', () => {
    expectToMatchSnapshot(
      <>
        <Button>Button</Button>
        <Button primary>Button primary</Button>
        <Button secondary>Button secondary</Button>
        <Button tertiary>Button tertiary</Button>
      </>,
    );
  });
});
