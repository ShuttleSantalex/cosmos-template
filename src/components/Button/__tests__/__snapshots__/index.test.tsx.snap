// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button matches snapshot 1`] = `
Array [
  <button onClick={[Function]}
  >
    Button
  </button>,
  <button onClick={[Function]}
  >
    Button primary
  </button>,
  <button onClick={[Function]}
  >
    Button secondary
  </button>,
  <button onClick={[Function]}
  >
    Button tertiary
  </button>,
]
`;
